src/App.scss

```diff
@@ -0,0 +1,31 @@
+.app {
+    display: flex;
+    width: 100vw;
+    align-items: center;
+    flex-direction: column;
+    .center-me {
+    }
+}
+
+.btn-nav {
+    background-color: rgb(162, 139, 184);
+    border-radius: 5px;
+    border-width: 1px;
+    color: lightblue;
+}
+
+.btn-nav:hover {
+    background-color: rgb(133, 77, 185);
+}
+
+div.header {
+    color: rgb(162, 139, 184);
```

Comment:
just a heads up, now that we have sass we can make color variables 🌈 

declare at the top of this file:
```scss
$color-name: rgb(162, 139, 184);
```

and use instead of value:
```scss
color: $color-name
```

Reply:
Hi, I updated and added $color-pale-purple.